cmake/mcuboot.cmake

```diff
@@ -111,6 +115,24 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
     --pad-header
     )
 
+  if(CONFIG_ZIGBEE)
+    set(zb_sign_cmd
+      ${PYTHON_EXECUTABLE}
+      ${NRF_DIR}/scripts/bootloader/zb_ota_sign.py
+      generate
+      --application ${PROJECT_BINARY_DIR}/app_signed.hex
```

Comment:
@mariuszpos What about other fields of OTA header format, for example `image_type`? Could it be provided somewhere else?